app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -1128,7 +1128,9 @@ public void onClick(View view) {
                 break;
 
             case R.id.report_site_issue:
-                SessionManager.getInstance().createSession(Source.MENU, SupportUtils.REPORT_SITE_ISSUE_URL.concat(getUrl()));
+                SessionManager.getInstance()
+                        .createSession(Source.MENU, SupportUtils.REPORT_SITE_ISSUE_URL
+                                .replace("%s", getUrl()));
```

Comment:
You may want to use [String.format()](https://developer.android.com/reference/java/lang/String.html#format(java.lang.String,%20java.lang.Object...)) for those replacements.